examples/grpc-debug/main.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"storj.io/storj/pkg/cfgstruct"
+	"storj.io/storj/pkg/provider"
+)
+
+var (
+	targetAddr = flag.String("target", "satellite.staging.storj.io:7777",
```

Comment:
don't wrap lines at 80

Reply:
:( but it makes side-by-side diffs so much better.

this is going to be a tough habit for me to break. i'll fix it here though